### PR TITLE
Expanded submission instructions in the 'Try it' section

### PR DESCRIPTION
**Pull request by ZapAnton**
Right now the `Try it!` section does not contain very clear instruction about what file you need to submit to pass the exercise and does not mention that you can submit multiple files, or re-submit your solution.

This PR tries to address this problem.

### CODE REVIEW

File: x/docs/md/track/TRY.md [hunk 1]
```diff
@@ -15,8 +15,15 @@ exercism configure --key=YOUR_EXERCISM_KEY
 ```
 
 When you've written a solution, submit it to the site.
-You'll have to configure the command-line client with your exercism API key before you can submit.
+The solution is a source file, in wich you have written your code
```
Comment (nickborromeo) [inline x/docs/md/track/TRY.md]: Maybe consider rewriting it to something like

```
When you've written a solution, a source file to pass the test suite of a problem, submit it to the site.
You'll have to configure the command-line client with your exercism API key before you can submit.
```

File: x/docs/md/track/TRY.md [hunk 2]
```diff
@@ -15,8 +15,15 @@ exercism configure --key=YOUR_EXERCISM_KEY
...
+to pass the test suite of a given problem, so the command to submit it will look like this:
 
 ```plain
-exercism submit PATH_TO_FILE
+exercism submit PATH_TO_YOUR_SOURCE_FILE
 ```
+
+If you have splitted your solution in several files or have used additional configuration files, submit them too.
```
Comment (nickborromeo) [inline x/docs/md/track/TRY.md]: I think we can just use `split` here instead of ` have splitted`. Also change `in` to `into`

**Comment by ZapAnton** [comment]
Done, @nickborromeo

File: x/docs/md/track/TRY.md [hunk 3]
```diff
@@ -14,9 +14,17 @@ In order to be able to submit your solution, you'll need to configure the client
...
+Submit the solution with the following command:
+
 ```plain
-exercism submit PATH_TO_FILE
+exercism submit PATH_TO_YOUR_SOURCE_FILE
 ```
+
+If you split your solution into several files or have used additional configuration files, submit them too.
+
+It is fine to submit incomplete solutions. If you are struggling with a problem, submit the solution you currently
```
Comment (nickborromeo) [inline x/docs/md/track/TRY.md]: I reviewed the documents that we already have and there is already prior art for submitting unfinished solutions. We can either link to that anchor here or we can repeat the text to keep the wording consistent. 

https://github.com/exercism/exercism.io#are-you-stuck-trying-to-solve-an-exercise

**Comment by ZapAnton** [comment]
Done, @nickborromeo. I repeated the text from existing documentation, since there is no quick way to get to the document you linked from the `Try it` section.

**Pull request merged**